bot_python_sdk/configuration_service.py

```diff
@@ -52,8 +51,11 @@ def activate(self):
             self.configuration_store.save(self.configuration)
 
     def generate_qr_code(self):
-        Logger.info(LOCATION, 'Generating QR Code for alternative pairing...')
-        device_information = self.configuration.get_device_information()
-        image = qrcode.make(json.dumps(device_information), image_factory=PymagingImage)
-        Store.save_qrcode(image)
-        Logger.success(LOCATION, 'QR Code successfully generated')
+        try:
+            Logger.info(LOCATION, 'Generating QR Code for alternative pairing...')
+            device_information = self.configuration.get_device_information()
+            image = qrcode.make(json.dumps(device_information), image_factory=PymagingImage)
+            Store.save_qrcode(image)
+            Logger.success(LOCATION, 'QR Code successfully generated')
+        except:
```

Comment:
Please raise the exception here too after logging

Reply:
exception has been raised in latest code